components/admin_console/group_settings/group_details/group_details.jsx

```diff
@@ -112,8 +117,15 @@ export default class GroupDetails extends React.PureComponent {
         this.setState({loadingTeamsAndChannels: false});
     }
 
+    onToggle = async (allowReference) => {
+        this.setState({allowReference});
+        await this.props.actions.patchGroup(this.props.groupID, {allow_reference: allowReference});
+    }
```

Comment:
Does `onToggle` need to `await` patchGroup? I wasn't seeing the response being used.

Reply:
👍 could drop the async / await on this method - unless theres something im missing

Reply:
agree.